week1/project/ecommerce/src/components/render-product.jsx

```diff
@@ -0,0 +1,17 @@
+import React  from 'react';
+
+
+const RenderProduct = ({product}) => {
+  return (
+    <div>
+        <img className = "product-img" src={product.image} alt = {product.title}/>
+        <h1>
+            Fake: {product.title}
+        </h1>
+    </div>
+  )
+}
```

Comment:
Two tips here:

1. You can do an "implicit return" here if you don't need to do anything else besides rendering within this component.
2. You can use a "React fragment" here instead of a div if you are not applying any styling. If you use a fragment you will not render a redundant `div` element 🙂 https://react.dev/reference/react/Fragment

Another small detail: the `alt` and `className` attributes have a space before defining the value but it needs to be together. Eventually it will be parsed correctly, but when viewing the code it looks like something is wrong.

```suggestion
const RenderProduct = ({product}) => (
    <>
        <img className="product-img" src={product.image} alt={product.title}/>
        <h1>
            Fake: {product.title}
        </h1>
    <>
  )
```